test/gallery.jl

```diff
@@ -6,37 +6,139 @@ using LinearAlgebra
 using SparseArrays
 using Test
 
-@testset "Gallery extra loading" begin
-    # Test if it is possible to load some extra gallery
-    @test (using GalleryWaveguide) == nothing
-end
+
+@testset "Gallery" begin
```

Comment:
Maybe that is important. My idea is that the gallery tests does not do much except loading the problem and doing on very simple computation. Typically the problems are also quite small, so I thought that the numbers may be mostly noise.

Reply:
It's not so important to me. My default would be to add it, since I don't know if we have any other tests that test gallery only, but rather load gallery and do something else, so if there's a bug introduced in gallery, this could be a useful data point to detect it. But if we think that this test will frequently vary in memory usage (e.g. frequent test or code changes), I agree it will just be noisy and better left out.